backend/infrahub/actions/tasks.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import annotations
+
+from infrahub_sdk.graphql import Mutation, Query
+from infrahub_sdk.node import InfrahubNode
+from prefect import flow
+from prefect.client.orchestration import get_client
+
+from infrahub.context import InfrahubContext  # noqa: TC001  needed for prefect flow
+from infrahub.core.constants import InfrahubKind
+from infrahub.core.registry import registry
+from infrahub.services import InfrahubServices  # noqa: TC001  needed for prefect flow
+from infrahub.trigger.models import TriggerType
+from infrahub.trigger.setup import setup_triggers
+from infrahub.workflows.catalogue import ACTION_RUN_GENERATOR
+from infrahub.workflows.utils import add_tags
+
+from .models import ActionTriggerRuleTriggerDefinition, EventGroupMember
+
+
+@flow(
+    name="add_node_to_group",
+    flow_run_name="Adding node={node_id} to group={group_id} on branch={branch_name}",
```

Comment:
```suggestion
    flow_run_name="Adding node={node_id} to group={group_id}",
```

Reply:
The flow_run_name is mainly used to display in Infrahub and in this case the name of the branch will also be visible so having it in the name of the task is redundant